problems/aramanim.py

```diff
@@ -0,0 +1,392 @@
+from manim import *
+import numpy as np
+
+def length(array: np.array):
+    array = np.array(array)
+    return np.matmul(array.transpose(), array) **(1/2)
+
+def normalized_vector(array: np.array):
+    array = np.array(array)
+    return (1/length(array)) * array
+
+
+def rotate2angle(array: np.array, alpha: float):
+    array = np.array(array)
+    rotation__matrix = [
+        [np.cos(alpha), -np.sin(alpha), 0],
+        [np.sin(alpha), np.cos(alpha), 0],
+        [0, 0, 1],
+    ]
+    return(np.matmul(rotation__matrix, array))
+
+def normal(array: np.array):
+    array = np.array(array)
+    return normalized_vector(rotate2angle(array, PI/2))
```

Comment:
Isn't this equivalent to just
`return normalized_vector([-array[1], array[0], array[2]])`
?

Reply:
You can skip this commemt